internal/search/backend/metered_searcher.go

```diff
@@ -55,13 +64,27 @@ func (m *meteredSearcher) Search(ctx context.Context, q query.Q, opts *zoekt.Sea
 		code = "error"
 	}
 
-	requestDuration.WithLabelValues(m.hostname, cat, code).Observe(time.Since(start).Seconds())
+	d := time.Since(start)
+	requestDuration.WithLabelValues(m.hostname, cat, code).Observe(d.Seconds())
 
 	tr.SetError(err)
 	if zsr != nil {
 		tr.LogFields(
 			log.Int("filematches", len(zsr.Files)),
-			log.Object("stats", &zsr.Stats),
+			log.Int64("rpc.latencyms", (d-zsr.Stats.Duration-zsr.Stats.Wait).Milliseconds()),
```

Comment:
nit: You capitalised MS below

```suggestion
			log.Int64("rpc.latencyMS", (d-zsr.Stats.Duration-zsr.Stats.Wait).Milliseconds()),
```

Reply:
yeah I only did that to match the camel case of the stats object. convention in opentracing is that this should actually be `rpc.latency_ms`. I think I'll update everything here to match convention.